src/client/src/app/modules/core/services/framework/framework.service.ts

```diff
@@ -118,4 +118,15 @@ export class FrameworkService {
     return _.get(this._channelData, 'defaultLicense');
   }
 
+  getSortedFilters(filters, type) {
```

Comment:
This function can be

`
getSortedFilters(filters, type) {
let sortedFilters = (type === 'gradeLevel' || _.lowerCase(type) === 'class') ? _.sortBy(filters, ['index', 'name') : _.sortBy(filters, 'name')
return sortedFilters;
}
`